### ISSUE TITLE
ROL missing ODEConstraint_TimeSimOpt.hpp

### ISSUE THREAD

**Issue by ibaned** [trilinos/Trilinos#3303]
@trilinos/rol 

## Expectations

ROL installs without error

## Current Behavior

```
CMake Error at packages/rol/adapters/mpi/src/cmake_install.cmake:49 (file):
  file INSTALL cannot find
  "/.../Trilinos/packages/rol/adapters/mpi/src/function/ODEConstraint_TimeSimOpt.hpp".
Call Stack (most recent call first):
  packages/rol/adapters/mpi/cmake_install.cmake:42 (include)
  packages/rol/cmake_install.cmake:72 (include)
  cmake_install.cmake:113 (include)
```

## Motivation and Context

Albany builds fail.

**Comment by dridzal** [comment]
@gregvw @eric-c-cyr , could you look into this?  Maybe the issue is in some cmake files still containing the removed ODEConstraint_TimeSimOpt.hpp?  It has been removed, right?  Thanks.

**Comment by bartgol** [comment]
I stumbled upon this as well. It appears commit b473a8f removed those files, but forgot to remove them from the CMakeLists.txt.

**Comment by ibaned** [comment]
Please approve #3308.

**Comment by mhoemmen** [comment]
I'm having troubles with this for SPARC's Trilinos build too.

**Comment by mhoemmen** [comment]
Yup, I can't build SPARC -- the ROL install fails.

**Comment by eric-c-cyr** [comment]
Yes, this is a known issue with a solution. Its being incorporated now.

**Comment by bartgol** [comment]
Wouldn't be useful if the autotester also ran 'make install' in addition to all tests? This kind of errors would be caught beforehand...

**Comment by mhoemmen** [comment]
@trilinos/framework There should be a Dashboard line for installation tests, no?  I don't think we've been doing those for PR testing -- perhaps that would be a good idea.

**Comment by bartlettroscoe** [comment]
This will be address by #2689 once we have time to get to that.  I think that Trilinos PR testing could do this as well.

**Comment by ibaned** [comment]
fixed by #3313
